date/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "date"
+version = "0.1.0"
+authors = ["Jonas Geschke <mail@yonny.de>"]
+build = "build.rs"
+edition = "2018"
+
+[dependencies]
+clap = { version = "^2.33.0", features = ["yaml", "wrap_help"] }
+chrono = { version = "0.4", features = ["serde"] }
```

Comment:
For version, I like to always put `~<version>` on packages with version minor than 1.0.0 so a patch update can be used, since patch update should not break API
While for packages that reached 1.0.0, `^<version>` so minor updates can also be used, since minor updates should not change API without proper deprecation